src/GafferUI/StandardConnectionGadget.cpp

```diff
@@ -186,6 +186,35 @@ void StandardConnectionGadget::doRenderLayer( Layer layer, const Style *style )
 		return;
 	}
 
+	// If we're missing the srcNodule, the source node is either hidden or we're dealing with an AuxiliaryConnection
+	if( !srcNodule() )
+	{
+		const Plug *srcPlug = dstNodule()->plug()->getInput();
+		const GraphGadget *graphGadget = parent<GafferUI::GraphGadget>();
+
+		if( const NodeGadget *srcNodeGadget = graphGadget->nodeGadget( srcPlug->node() ) )
+		{
+			if( srcNodeGadget->getVisible() )
+			{
+				const Box3f b = srcNodeGadget->transformedBound();
+				const Box2f srcNodeFrame( V2f( b.min.x, b.min.y ), V2f( b.max.x, b.max.y ) );
+
+				const M44f m = dstNodule()->fullTransform( srcNodeGadget );
+				const V3f v = V3f( 0 ) * m;
+				const V2f dstPosition = V2f( v.x, v.y );
+
+				const NodeGadget *dstNodeGadget = graphGadget->nodeGadget( dstNodule()->plug()->node() );
+				const V3f t = dstNodeGadget->noduleTangent( dstNodule() );
+				const V2f dstTangent = V2f( t.x, t.y );
+
+				Style::State state = srcNodeGadget->getHighlighted() || dstNodeGadget->getHighlighted() ? Style::HighlightedState : Style::NormalState;
```

Comment:
Again, why duplicate logic that already exists below (line 220 onwards)?

There's a related question here I think. Currently it's possible to start a drag with one of these auxiliary connections (only from the dst end as far as I can tell), but there's no visual indication that it's happening at all, until you drop into empty space and the connection is deleted. Do we want to allow that? To me it seems like it could be potentially useful, and I think providing the visual feedback is simply a case of removing the duplicate logic from in here.

I suspect the Expression node might not be super happy if the drag allowed an output to be connected into two places, but that will be fixed by the upcoming "OSLCode-stylee" expressions we've been talking about.